package.json

```diff
@@ -13,42 +13,42 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~10.0.9",
+    "@angular/animations": "~9.1.12",
     "@angular/cdk": "^8.0.0",
-    "@angular/common": "~10.0.9",
-    "@angular/compiler": "~10.0.9",
-    "@angular/core": "~10.0.9",
-    "@angular/forms": "~10.0.9",
-    "@angular/platform-browser": "~10.0.9",
-    "@angular/platform-browser-dynamic": "~10.0.9",
-    "@angular/router": "~10.0.9",
-    "devextreme": "20.1-stable",
-    "devextreme-angular": "20.1-stable",
-    "devextreme-schematics": "^1.2.3",
-    "rxjs": "~6.5.5",
-    "tslib": "^2.0.0",
-    "zone.js": "~0.10.3"
+    "@angular/common": "~9.1.12",
+    "@angular/compiler": "~9.1.12",
+    "@angular/core": "~9.1.12",
+    "@angular/forms": "~9.1.12",
+    "@angular/platform-browser": "~9.1.12",
+    "@angular/platform-browser-dynamic": "~9.1.12",
+    "@angular/router": "~9.1.12",
+    "devextreme": "^20.2.4",
+    "devextreme-angular": "^20.2.4",
+    "devextreme-schematics": "file:../../../../../devextreme-schematics",
```

Comment:
it seems `npm i ` will fail with this

Reply:
Replaced to "devextreme-schematics": "^1.2.6"